### PR TITLE
android-ci.yml

### PR DESCRIPTION
**Pull request by Lindet94**
CI configuration for the android app.

### CODE REVIEW

**Comment by platisd** [comment]
So, the build fails, how would you fix it?

**Review by platisd — COMMENTED** [review]
Why the empty directory? Why not try to understand what the workflow you have committed does and modify that instead?

**Comment by Lindet94** [comment]
> Why the empty directory? Why not try to understand what the workflow you have committed does and modify that instead?

Right, just wanted to see what would happen after reading something online. What I did now was to basically comment out the tests for the SmartcarController as I'm not sure how to work with this yet, as well as the instrumented tests. Does it look alright for now or should I do some other changes?

**Review by platisd — CHANGES_REQUESTED** [review]
No, it doesn't look alright. You literally disabled the CI checks to make them pass. :sweat_smile:

**Comment by platisd** [comment]
You didn't have to change the directory you are using just so you can copy paste the existing configuration, you could adjust it to your project structure.

**Comment by Lindet94** [comment]
> You didn't have to change the directory you are using just so you can copy paste the existing configuration, you could adjust it to your project structure.

Yeah I know but somehow it didn't seem to accept the change so decided to just change the directory instead. I guess I did something wrong there.

**Comment by Lindet94** [comment]
> No, it doesn't look alright. You literally disabled the CI checks to make them pass. 😅

Big brain move! :) I'm not exactly a quick learner but I don't give up easily... 

Anyway, looks like we are getting somewhere here now?

**Comment by platisd** [comment]
Yes it's good now, but not a big fan of using the same directory as the sample app. Can you change it and use `android/` like you originally meant to?

File: .github/workflows/android.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+name: Android CI
+
+on: [push, pull_request]
+
+defaults:
+  run:
+    working-directory: android
```
Comment (platisd) [inline .github/workflows/android.yml]: If you want set a working directory you should do this one the specific step you are interested in **or** simply `cd android` before you run the gradle command.

**Review by platisd — CHANGES_REQUESTED** [review]

File: .github/workflows/android.yml (hunk 1 above)
Comment (Lindet94) [reply .github/workflows/android.yml]: Yeah it seems to work the same both ways?

How do I use chmod +x gradlew in a proper way? Having a hard time finding a resource explaining this commad. Where do I use it?

Comment (Lindet94) [reply .github/workflows/android.yml]: Nevermind, Think I got it.

**Review by platisd — APPROVED** [review]
(ﾉ◕ヮ◕)ﾉ*:･ﾟ✧
Looks good now

**Pull request merged**